lib/discord.go

```diff
@@ -213,6 +265,48 @@ func GetBotUser(token string) (*BotUserResponse, error) {
 }
 
 func doDiscordReq(ctx context.Context, path string, method string, body io.ReadCloser, header http.Header, query string) (*http.Response, error) {
+	identifier := ctx.Value("identifier")
+	if identifier == nil {
+		// Queues always have an identifier, if there's none in the context, we called the method from outside a queue
+		identifier = "Internal"
+	}
+
+	logger.Info(method, path+"?"+query)
+
+	identifierStr, ok := identifier.(string)
+
+	if ok {
+		// Check endpoint cache
+		if endpointCache[identifierStr] != nil {
```

Comment:
Can we put this (and the other code path related to caching) behind an env var? I'm not comfortable surprising people with a change like this, on the next major we can possibly have it default to true.

Reply:
OK